### ISSUE TITLE
Cannot read property 'get' of null

### ISSUE THREAD

**Issue by YevgenSereda** [avocode/avocode#1651]
* **Version**: 2.15.5

* **Web/Desktop**: Desktop

* **Platform**: win32

* **Avocode UID**: 192913
---

```
TypeError: Cannot read property 'get' of null
    at Function.value (file:///C:/Users/ievgen.sereda/AppData/Local/avocode/app-2.15.5/resources/app.asar/dist/avocode.min.js:1:46406)
    at Function.value (file:///C:/Users/ievgen.sereda/AppData/Local/avocode/app-2.15.5/resources/app.asar/dist/avocode.min.js:1:49311)
    at file:///C:/Users/ievgen.sereda/AppData/Local/avocode/app-2.15.5/resources/app.asar/dist/avocode.min.js:1:438662
    at C:\Users\ievgen.sereda\AppData\Local\avocode\app-2.15.5\resources\app.asar\node_modules\immutable\dist\immutable.js:4364:24
    at List.__iterate (C:\Users\ievgen.sereda\AppData\Local\avocode\app-2.15.5\resources\app.asar\node_modules\immutable\dist\immutable.js:2208:13)
    at List.every (C:\Users\ievgen.sereda\AppData\Local\avocode\app-2.15.5\resources\app.asar\node_modules\immutable\dist\immutable.js:4363:12)
    at Function.value (file:///C:/Users/ievgen.sereda/AppData/Local/avocode/app-2.15.5/resources/app.asar/dist/avocode.min.js:1:438621)
    at
```

**Comment by jakubzitny** [comment]
Hi @YevgenSereda, thanks for reporting this error. It is a bug, we already have it in our bug tracking system, so we'll try to fix it for the next release. Let us know if you have any more problems with Avocode 😉

**Comment by jakubzitny** [comment]
Hi, this has been fixed in latest Avocode `2.16.1`, please update and let us know if you have any more problems :wink:.
